FWCore/Framework/src/SubProcess.cc

```diff
@@ -210,6 +210,62 @@ namespace edm {
 
   SubProcess::~SubProcess() {}
 
+  std::vector<ModuleProcessName> SubProcess::keepOnlyConsumedUnscheduledModules(bool deleteModules) {
+    ServiceRegistry::Operate operate(serviceToken_);
```

Comment:
Why is the services turned on here?

Reply:
I probably thought that it would be needed because in the current code the services are turned on before
```cpp
schedule_->convertCurrentProcessAlias(processConfiguration_->processName());
```
Removing it did not cause any framework unit test to fail, so I'll remove it. Thanks.